packages/public/server/src/module/Entity/src/Entity/Controller/EntityController.php

```diff
@@ -63,4 +65,32 @@ public function checkResponse(ResponseCollection $response)
         }
         return true;
     }
+
+    public function unrevisedAction()
+    {
+        $revisions = $this->getEntityManager()->findAllUnrevisedRevisions()->getIterator();
+
+        $revisions->uasort(function ($revisionA, $revisionB) {
+            $timestampA = $revisionA->getTimestamp()->getTimestamp();
+            $timestampB = $revisionB->getTimestamp()->getTimestamp();
+
+            return $timestampB - $timestampA;
+        });
+
+        $revisionsBySubject = array();
+
+        foreach ($revisions as $revision) {
+            $normalizedRevision = $this->getNormalizer()->normalize($revision);
```

Comment:
If you normalize this here, instead of in the view, the disadvantage is that you loose some accesses to the revisions internals. #137 asked for the `changes` field to be displayed in a column, like we do in the revision history of the entity (see https://github.com/serlo/serlo.org/blob/master/packages/public/server/src/module/Ui/templates/entity/repository/history.twig#L51). That is not possible with the normalized element. I think passing the raw revisions to the view would be ok here.